nixops/backends/hetzner.py

```diff
@@ -223,13 +247,27 @@ def _bootstrap_rescue_for_existing_system(self):
         self.run_command("cat >> /etc/motd", stdin_string=fullmsg)
         self.log_end("done.")
 
-    def _bootstrap_rescue(self, install, partitions):
+    def _bootstrap_rescue(self,
+                          install,
+                          partitions,
+                          partitioning_script=None,
+                          mount_script=None,
+                          fs_info=None):
         """
         Bootstrap everything needed in order to get Nix and the partitioner
         usable in the rescue system. The keyword arguments are only for
         partitioning, see reboot_rescue() for description, if not given we will
         only mount based on information provided in self.partitions.
+
+        Exactly one of `partitions` and `partitioning_script` must be given as
+        non-None value.
```

Comment:
I'd check this via the NixOS module system as well.

Reply:
Yes, but not a reason not to merge, is it?